audonnx/core/model.py

```diff
@@ -223,17 +224,31 @@ def __call__(
         the output of that node is returned.
         Otherwise a dictionary with outputs of all nodes is returned.
 
+        If ``concat`` is set to ``True``,
+        the output of the requested nodes is concatenated
+        and a single array is returned.
+        This requires that the number of dimensions,
+        the position of dynamic axis,
+        and all dimensions except the last non-dynamic axis
```

Comment:
I guess we will never need the feature to select an axis, right?
I would also not be in favor of it, as it would mean adding another argument to select the axis.

Reply:
Yes, and we also already assume that the labels always apply to the last non-dynamic axis.